javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,63 @@
+console.log("Script loaded");
+
+let products = getAvailableProducts();
+
+
+function renderProducts(products) {
+    // your code here
+    const ul = document.querySelector("ul");
+    products.forEach(product => {
+        const li = document.createElement("li");
+        li.innerHTML = `<h2>${product.name}</h2><span>price: ${product.price}</span><br><span>Rating: ${product.rating}</span>`;
+        ul.appendChild(li);
+    });
+}
+renderProducts(products);
```

Comment:
How about filtering the products that need to be displayed _both_ by text input (if there is any) and price input (also if any)? (With some other modifications in event listeners). 
Then both of your filters would work at once.

Reply:
Solved